src/database/queries/canonicalFields/fetchAll.sql

```diff
@@ -0,0 +1,7 @@
+SELECT
+  cf.id as "id",
+  cf.label as "label",
+  cf.short_code as "shortCode",
+  cf.data_type as "dataType",
+  cf.created_at as "createdAt"
+FROM canonical_fields as cf;
```

Comment:
Thanks for humoring me here too with the terser table alias. It looks nicer to me, at least!

Reply:
I agree!